xen/arch/arm/tee/ffa_private.h

```diff
@@ -430,4 +432,12 @@ static inline int32_t ffa_rx_release(void)
     return ffa_simple_call(FFA_RX_RELEASE, 0, 0, 0, 0);
 }
 
+static inline bool ffa_fw_support_fid(uint32_t fid)
```

Comment:
We should ideally have a verb in this function name. How about `ffa_fw_supports_fid()`?

Reply:
ack